platform/util/platform-util/src/main/java/org/codice/ddf/platform/util/RandomNumberGenerator.java

```diff
@@ -22,11 +22,11 @@
 
 public class RandomNumberGenerator {
```

Comment:
@bradh that is a good question/point. I don't believe this is public API, but somebody can correct me if I'm wrong.